mediapipe/calculators/ovms/openvinoinferencecalculatoroptions.cc

```diff
@@ -0,0 +1,249 @@
+//*****************************************************************************
+// Copyright 2023 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+#include <unordered_map>
+
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wdeprecated-declarations"
+#include "mediapipe/framework/calculator_contract.h"
+#include "mediapipe/calculators/ovms/openvinoinferencecalculator.pb.h"
+#pragma GCC diagnostic pop
+#include "mediapipe/calculators/ovms/openvinoinferencecalculatoroptions.h"
+
+namespace mediapipe {
+using std::endl;
```

Comment:
This will be included in all units that include this header. Remove this and move to .cpp file where it is really needed (is it?)